index.html

```diff
@@ -0,0 +1,359 @@
+          <div class="col-4">
+            <div class="row">
+              <div class="col-12"><span class="icon-search"></span></div>
+              <div class="col-12"><p>Search Places</p></div>
+            </div>
+          </div>
+          <div class="col-4">
+            <div class="row">
+              <div class="col-12"><span class="icon-check-circle"></span></div>
+              <div class="col-12"><p>Select Places</p></div>
+            </div>
+          </div>
+          <div class="col-4">
+            <div class="row">
+              <div class="col-12"><span class="icon-thumbs-up"></span></div>
+              <div class="col-12"><p>Book & Enjoy</p></div>
+            </div>
+          </div>
+          <object
+            class="dotted_line"
+            data="img/Line.svg"
+            type="image/svg+xml"
+          ></object>
+        </div>
+      </div>
+    </section>
+    <section class="firstSpain">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-7 col-xl-6 col-sm-12 firstSpain_block">
+            <div class="row">
+              <div class="col-12"><p>Spain</p></div>
+              <div class="col-12"><h3>Grand Spain Madrid</h3></div>
+              <div class="col-12">
+                <p class="firstSpain_p">
+                  But I must explain to you how all this mistaken idea of
+                  account to denouncing pleasure and praising pain was account
+                  born and I account will give you a complete account of the
+                  system
+                </p>
+              </div>
+              <div class="col-12 firstSpain_button">
+                <a href="#" class="btn-1">book now</a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="secondSpain">
+      <div class="container">
+        <div class="row">
+          <div class="col-6"></div>
+          <div class="col-lg-7 col-xl-6 col-sm-12 secondSpain_block">
+            <div class="row">
+              <div class="col-12"><p>Spain</p></div>
+              <div class="col-12"><h3>Grand Spain Madrid</h3></div>
+              <div class="col-12">
+                <p class="secondSpain_p">
+                  But I must explain to you how all this mistaken idea of
+                  account to denouncing pleasure and praising pain was account
+                  born and I account will give you a complete account of the
+                  system
+                </p>
+              </div>
+              <div class="col-12 secondSpain_button">
+                <a href="#" class="btn-1">book now</a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="tourists">
+      <div class="container">
+        <div class="row">
+          <div class="col-12">
+            <h2>What Our Tourists Say</h2>
+          </div>
+        </div>
+        <div class="row">
```

Comment:
Нужно убрать отступы между колонками
![image](https://user-images.githubusercontent.com/34746432/117668511-aecc9100-b1ae-11eb-94b1-ac0d69eda846.png)

```suggestion
        <div class="row g-0">
```